ember-toucan-core/src/-private/components/error.hbs

```diff
@@ -1,11 +1,7 @@
-<div
-  class="type-xs-tight text-critical mt-1.5 flex
-    {{if this.hasMoreThanOneError 'items-start' 'items-center'}}"
-  ...attributes
->
+<div ...attributes>
   {{! TODO: https://github.com/CrowdStrike/ember-toucan-core/issues/49 - Placeholder icon }}
   <svg
-    class="mr-1 h-3 w-3"
+    class="mr-1 -mt-1 inline-block h-3 w-3"
```

Comment:
Note: added a negative margin top because the SVG did not look vertically centered. Other things I tried

- align-middle (both on this div and the parent div)

Open to suggestions on this however!